### PR TITLE
Add test for OnlyModule option in resolve.alias

### PR DESCRIPTION
**Pull request by jeffin143**
**What kind of change does this PR introduce?**
Add test 

  
**Did you add tests for your changes?**

test for  `onlyModule`: `Redirect only exact matching request.` option in resolve.alias
**Does this PR introduce a breaking change?**
No

**What needs to be documented once your changes are merged?**
None

### CODE REVIEW

**Comment by jeffin143** [comment]
No clue why lint and basic fails

**Comment by sokra** [comment]
> No clue why lint and basic fails

you did not run `yarn` before the tests

File: test/__snapshots__/StatsTestCases.test.js.snap [hunk 1]
```diff
@@ -835,9 +835,9 @@ external \\"test\\" 42 bytes [built]"
 `;
 
 exports[`StatsTestCases should print correct stats for filter-warnings 1`] = `
-"Hash: 7763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca17763b48a6d4533935ca1
```
Comment (sokra) [inline test/__snapshots__/StatsTestCases.test.js.snap:838]: should be unchanged

File: types.d.ts [hunk 2]
```diff
@@ -161,14 +161,14 @@ declare interface Argument {
 	description: string;
 	simpleType: "string" | "number" | "boolean";
 	multiple: boolean;
-	configs: ArgumentConfig[];
```
Comment (sokra) [inline types.d.ts:164]: should be unchanged

File: test/__snapshots__/StatsTestCases.test.js.snap (hunk 1 above)
Comment (jeffin143) [reply test/__snapshots__/StatsTestCases.test.js.snap]: Weird , No clue why test were failing 

All green

**Pull request merged**